testsuite/MDAnalysisTests/analysis/test_density.py

```diff
@@ -298,7 +381,7 @@ def test_ValueError_noatomgroup(self, universe):
 
     def test_warn_results_deprecated(self, universe):
         D = density.DensityAnalysis(
-                universe.select_atoms(self.selections['static']))
+            universe.select_atoms(self.selections['static']))
```

Comment:
The visual indentation here is probably worse? Is this the result of an auto-formatter? Personally I'd prefer we stick with the previous formatting for this line.